packages/rollup-plugin-minify-html-literals/README.md

```diff
@@ -0,0 +1,82 @@
+# @lit/rollup-plugin-minify-html-literals
+
```

Comment:
We need some more intro text here an any standard Labs verbiage we use.

Reply:
Do we consider this plugin labs? Then the package scope should be updated to `@lit-labs` and this moved to `packages/labs`.

Reply:
ooh, good point. Yeah, I was assuming it was labs, but didn't look at the rest of the package for that. Maybe I'm wrong though. We should discuss.